src/core/field-text-check.js

```diff
@@ -53,14 +53,11 @@ export function checkTextField(fieldName, fieldText, allowedLinks, optionalField
         return result;
 
     // Create our more detailed location string by prepending the fieldName
-    let ourAtString = ` in '${fieldName}'`;
-    if (optionalFieldLocation) {
-        if (optionalFieldLocation[0] !== ' ') ourAtString += ' ';
-        ourAtString += optionalFieldLocation;
-    }
+    let ourLocation = optionalFieldLocation;
+    if (ourLocation && ourLocation[0] !== ' ') ourLocation = ` ${ourLocation}`;
```

Comment:
In testing, I found that fieldName is missing from location field in error messages.  It seems that it should be added back in here.  Otherwise you get a message like `Unexpected doubled . characters`, but still have to go looking through all the fields of the tsv to find it.

Reply:
No @PhotoNomad0, it doesn't need to be added back into the `location` field of the notices, but if it's missing from the error messages, then I must have missed passing through the `rowID` field somewhere and need to find where that is.

Reply:
I think we need to accept this PR so we can progrress, since it was a fairly large refactoring. I will still be working on improving those and other notices, but those small fixes hopefully will no longer change the API.